src/NuGet.CommandLine/MsBuildUtility.cs

```diff
@@ -26,48 +27,82 @@ public static bool IsMsBuildBasedProject(string projectFullPath)
             return _msbuildExtensions.Contains(Path.GetExtension(projectFullPath));
         }
 
-        public static IEnumerable<string> GetProjectReferences(string projectFullPath)
+        public static IEnumerable<string> GetProjectReferences(string msbuildPath, string projectFullPath)
         {
-            var projectDirectory = Path.GetDirectoryName(projectFullPath);
-            var project = new ProjectInstance(projectFullPath);
+            if (string.IsNullOrEmpty(msbuildPath))
+            {
+                msbuildPath = Path.Combine(
+                    Environment.GetFolderPath(Environment.SpecialFolder.ProgramFilesX86),
```

Comment:
Does this work on an X86 machine? If this doesn't exist it should use the non-x86 var.